src/shared/tpm2-util.h

```diff
@@ -5,6 +5,10 @@
 #include "macro.h"
 #include <stdbool.h>
 
+typedef enum {
```

Comment:
`typedef enum tpm2_flags {` - also no need for the systemd_ prefix

Reply:
Looking through the codebase, I think i should use CamelCase for consistency, too.

Reply:
yes